### DISCUSSION TITLE
Increase acceleration for UR

### DISCUSSION THREAD

**Discussion by KonradJuenger** [visose/Robots#258] [General]
Hi, This is more of a feature request.

By default, “Robots” has an acceleration of 1, which restricts how fast the robot can move. You can use a _find/replace text_ element after the output to increase the acceleration, but it would be better if this option was available in the “create speed” component. See the difference between the default and the improved acceleration below:

https://github.com/visose/Robots/assets/76114767/428e0405-6a06-441a-8a4b-693dcdb3e8c8

https://github.com/visose/Robots/assets/76114767/bb3f6df8-0773-4da5-a6bf-d89a39d5a9a3

![image](https://github.com/visose/Robots/assets/76114767/9d1ddf39-14f7-4ac8-b761-05a086350431)

[acceleration_ur.zip](https://github.com/visose/Robots/files/13862193/acceleration_ur.zip)

**Comment by visose** [comment]
You can use a Python/C# scripting component to change the accel value of a speed parameter, but I'll make it available as an input in the next version, this will be useful for Franka Emika too.

**Comment by visose** [comment]
Version 1.6.7 is now available with a new Speed component that has two accel inputs for translation and axes. Translation defaults to 2500 mm/s2 which I read is recommended to set below this value for linear motions. Axis defaults to 4PI, it's recommended to set below 800 deg/s2.

For Franka Emika, Axis accel can be used for the dynamic_rel value, assuming 4PI is the maximum value (set to 2PI for a dynamic_rel value of 0.5).

The previous Speed component is set as obsolete and will use the old default accel values. The text to Speed cast will also use the old default values to not break backwards compatibility.

Feel free to reopen this issue if something is not right.
